org.eclipse.mylyn.releng/multipass/inventory.yml

```diff
@@ -37,7 +38,7 @@ all:
       ansible_user: ubuntu
       ansible_ssh_common_args: "-o StrictHostKeyChecking=no -o ControlMaster=no -o ControlPath=none"
       ansible_ssh_private_key_file: first_setup/ubuntu_key
-      default_etc_hosts: "mylyn-local traefik.mylyn.local mylyn-local.mylyn.local mylyn.local jenkins.mylyn.local hello.mylyn.local artifactory.mylyn.local gitlab.mylyn.local"
+      default_etc_hosts: "mylynmstr01 mylyn.local traefik.mylyn.local jenkins.mylyn.local hello.mylyn.local artifactory.mylyn.local gitlab.mylyn.local"
```

Comment:
I had problems with ubuntu about mylynmstr01 being associated with the external ip address. 
I had to change it to be like mylynadm01
```
10.209.22.134 mylynadm01
10.209.22.130 mylynmstr01
192.168.0.160  mylyn.local ...
```